src/com/exadel/bootcamp/CashMachine.java

```diff
@@ -19,9 +18,27 @@ public String getName() {
         return this.name;
     }
 
-    public Map getBanknotes() {
-        return banknotes;
+    public void giveClientCash(int withdrawCash) {
+        while (withdrawCash > 0) {
```

Comment:
This method works. that's allright. But is not that readable.
1) three levels of nested code is no good. Usually, two levels max is allowed (there can be some third, but only if it is a very simple `if`, or something of that sort
2) there's no visible result. It's hard to see when this method finishes. And with what outcome it finishes. It does something.. does.. does.. then breaks. We need a visible marker, like `return something` -- and we see what is being returned. Or a statement after the loop that summarises what happened (maybe, logs it).

Consider converting it into a function returning how much money was actually given out
Consider extracting the inner `for` into a separate method.
(this will not affect approvement of the current PR)